calico_rkt/logutils.py

```diff
@@ -0,0 +1,17 @@
+#!/bin/python
+import logging
+import os
+
+LOG_DIR = '/var/log/calico/rkt-plugin'
+
+def configure_logger(logger, log_dir=None):
```

Comment:
Alternatively since we have similar code elsewhere, there's an argument that it could go in libcalico and we import it from there.